test/config/config-defaults.yaml

```diff
@@ -0,0 +1,90 @@
+# Copyright 2019 The Knative Authors
```

Comment:
For those configmaps, looks like we directly copy them from serving repo. Could we import them into vendor, and use them directly from vendor?

Reply:
I will follow up since vendoring config/ will require a change in knative.dev/serving as well.

Also, these configmaps are actually empty.